packages/persist-state/src/lib/storage.ts

```diff
@@ -33,9 +33,26 @@ function createStorage(storage: Storage | undefined): StateStorage | undefined {
   };
 }
 
-export const localStorageStrategy = createStorage(
-  typeof localStorage !== 'undefined' ? localStorage : undefined
-)!;
-export const sessionStorageStrategy = createStorage(
```

Comment:
Please add a comment describing why we need the try catch

Reply:
I've added a comment 👍🏻